spec/controllers/api/v1/photos_controller_spec.rb

```diff
@@ -3,20 +3,46 @@
 module Api
   module V1
     describe PhotosController do
+      render_views
+
       let(:user) { create(:user) }
-      let!(:cake) { create(:creation) }
+      let!(:cake) { create(:creation, user: user) }
 
-      describe "#index" do
-        before :each do
-          request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Token.encode_credentials(user.authentication_token)
-          user.creations << cake
-        end
+      before :each do
+        request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Token.encode_credentials(user.authentication_token)
+      end
 
+      describe "#index" do
         it 'loads the cakes photos' do
           xhr :get, :index, cake_id: cake.id
           expect(assigns(:photos)).to match_array(cake.photos)
         end
       end
+
+      describe "#show" do
+        let(:photo) { create(:photo, imageable: cake) }
+
+        it 'loads the photo' do
+          xhr :get, :show, cake_id: cake.id, id: photo.id
+          expect(assigns(:photo)).to eql(photo)
+        end
+      end
+
+      describe "#create" do
+        let(:file) { fixture_file_upload('images/example.png', 'image/png') }
+
+        it 'attaches a new photo to a cake' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.